src/xrGame/alife_object.cpp

```diff
@@ -22,72 +22,80 @@ void CSE_ALifeObject::spawn_supplies(LPCSTR ini_string)
     if (!xr_strlen(ini_string))
         return;
 
+    luabind::functor<bool> funct;
+	if (GEnv.ScriptEngine->functor("ai_stalker.CSE_ALifeObject_spawn_supplies", funct))
+	{
+		if (funct(this, ID, ini_string))
+			return;
+	}
```

Comment:
Tabs should be replaced with spaces :)
```suggestion
    if (GEnv.ScriptEngine->functor("ai_stalker.CSE_ALifeObject_spawn_supplies", funct))
    {
        if (funct(this, ID, ini_string))
            return;
	}
```